src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -590,9 +614,19 @@ def is_multi_npu():
     return (num_npus > 1)
 
 
+def is_single_voq():
+    chassis_db_conf_file_path = get_chassis_db_conf_file_path()
+    if chassis_db_conf_file_path is None:
+        return True
+
+    return False
```

Comment:
Function name is_single_voq() misleading based on the function contents.  The checking inside the function only identify there is no "chassisdb.conf" file.   All pizzabox (include the Non-VOQ platforms ) which doesn't chassisdb.conf will return True for this function.